### ISSUE TITLE
Select dirlist when entering a directory.

### ISSUE THREAD

**Issue by mowgli** [BestImageViewer/geeqie#356]
Patch to make the first dir list entry ("..") selected upon entering a directory. This allows one to quickly browse in subdirectories (move around with keys, Enter+Enter to check one subdir, etc.)

Reported by: unqwist

**Comment by mowgli** [comment]
Original patch:
```patch
diff -ur geeqie-1.0.orig/src/view_dir_list.c geeqie-1.0/src/view_dir_list.c
--- geeqie-1.0/src/view_dir_list.c	2010-02-17 21:21:19.000000000 +0000
+++ geeqie-1.0/src/view_dir_list.c	2012-07-08 15:44:21.000000000 +0000
@@ -311,7 +311,7 @@
 
 	ret = vdlist_populate(vd, TRUE);
 
-	if (old_path)
+	if (1)
 		{
 		/* scroll to make last path visible */
 		FileData *found = NULL;
@@ -321,13 +321,13 @@
 		while (work && !found)
 			{
 			FileData *fd = work->data;
-			if (strcmp(old_path, fd->name) == 0) found = fd;
+			if (!old_path || strcmp(old_path, fd->name) == 0) found = fd;
 			work = work->next;
 			}
 
 		if (found) vdlist_scroll_to_row(vd, found, 0.5);
 
-		g_free(old_path);
+		if (old_path) g_free(old_path);
 		return ret;
 		}
 
```

**Comment by caclark** [comment]
Implemented in commit 7831e6372 at geeqie.org
